### DISCUSSION TITLE
Bun v1.1.3

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#10075] [Announcements]
To install Bun v1.1.3:
```bash
curl https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

Windows:
```bash
powershell -c "irm bun.sh/install.ps1|iex"
```

To upgrade to Bun v1.1.3:
```bash
bun upgrade
```

#### Thanks to 5 contributors!

- [@Bellisario](https://github.com/Bellisario)
- [@Jarred-Sumner](https://github.com/Jarred-Sumner)
- [@jprinaldi](https://github.com/jprinaldi)
- [@tomerh2001](https://github.com/tomerh2001)
- [@yoavbls](https://github.com/yoavbls)

**Full Changelog**: https://github.com/oven-sh/bun/compare/bun-v1.1.2...bun-v1.1.3

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.1.3'>Bun v1.1.3</a>.</em>

**Comment by 8xu** [comment]
_**wake up bae, [Bun v1.1.3](https://github.com/oven-sh/bun/releases/tag/bun-v1.1.3) dropped!**_ 🙂
